### DISCUSSION TITLE
Loading BraTS-20 dataset for Tumor Segmentation

### DISCUSSION THREAD

**Discussion by mfaizan-ai** [Project-MONAI/MONAI#3955] [Q&A]
I have a BraTS-20 dataset for brain tumor segmentation. I want to load this dataset for training. I have gone through this [tutorial](https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/brats_segmentation_3d.ipynb), however, in this tutorial author uses DecathlonDataset for loading the BraTS-16 and BraTS-17 datasets. I need to load BraTS-20, I have searched through the issues on this forum but I have not found the required tutorial. I want to load the BraTS-20 dataset using data loaders. Anyone, please help me to sort this out.

**Comment by Nic-Ma** [comment]
Hi @faizan1234567 ,

Thanks for your interest.
I think you can refer to the dataset and dataloader in this tutorial:
https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/spleen_segmentation_3d.ipynb

Thanks.

**Comment by WenqingZong** [comment]
Hello, I'm doing transfer learning so wondering if it's possible to only load one modality from BraTS dataset? Thanks in advance!

**Reply by Nic-Ma** [reply]
Hi @dongyang0122 ,

Could you please help share some best practices about this question?

Thanks in advance.

**Comment by dongyang0122** [comment]
Hi @WenqingZong, you can customize a simple transform via [Lambdad](https://docs.monai.io/en/stable/transforms.html#monai.transforms.Lambdad) to select input channels of images. For example, 
`Lambdad(keys='image', func=lambda x: x[0:1, :, :])`.
Once you select the channel, you may need [EnsureChannelFirstd](https://docs.monai.io/en/stable/transforms.html#monai.transforms.EnsureChannelFirstd) to make sure the input data is 4-D before batching.

**Reply by WenqingZong** [reply]
Hi @dongyang0122 , thank you for your reply, I got another question on the dataset. I know BraTS contains 4 modalities, (T1, T1ce, T2, FLAIR), and I currently have a batch of shape (BatchSize, Modality, Height, Width, Depth), I'd like to know which modality index corresponds to which actual modality?

**Reply by WenqingZong** [reply]
It's important as my transfer learning especially focuses on the direction from T1ce to T2.
